tests/testthat/test_combine.R

```diff
@@ -35,18 +35,18 @@ test_that("combine", {
     expect_equal(assayNames(cons_adj), 
         c("binary", "group", "formula", "mass", 
             "clr_coef", "aracne_coef", "pearson_coef", "pearson_pvalue", 
-            "spearman_coef", "spearman_pvalue", "bayes_coef", "consensus",
-            "combine_binary", "combine_group", "combine_formula", 
-            "combine_mass"))
-    expect_equal(sum(assay(cons_adj, "combine_binary"), na.rm = TRUE), 4)
-    expect_equal(as.vector(assay(cons_adj, "combine_binary")[1, ]), 
-        c(NA, 0, 0, 0, 1, 0, 0))
-    expect_equal(as.vector(assay(cons_adj, "combine_group")[1, ]), 
-        c(NA, "", "", "", "Monosaccharide (-H2O)", "", ""))
-    expect_equal(as.vector(assay(cons_adj, "combine_formula")[1, ]), 
-        c(NA, "", "", "", "C6H10O5", "", ""))
-    expect_equal(as.vector(assay(cons_adj, "combine_mass")[1, ]), 
-        c(NA, "", "", "", "162.0528234315", "", ""))
+          "spearman_coef", "spearman_pvalue","ggm_coef", "ggm_pvalue",
+          "bayes_coef", "consensus","combine_binary", "combine_group", 
+          "combine_formula",  "combine_mass"))
+    expect_equal(sum(assay(cons_adj, "combine_binary"), na.rm = TRUE), 2)
+    expect_equal(as.vector(assay(cons_adj, "combine_binary")[5, ]), 
+        c(0, 0, 0, 0, NA, 1, 0))
+    expect_equal(as.vector(assay(cons_adj, "combine_group")[5, ]), 
+        c("", "", "", "", NA, "Malonyl group (-H2O)", ""))
+    expect_equal(as.vector(assay(cons_adj, "combine_formula")[5, ]), 
+        c( "", "", "","", NA, "C3H2O3",  ""))
+    expect_equal(as.vector(assay(cons_adj, "combine_mass")[5, ]), 
+        c("", "", "", "", NA,  "86.0003939305", ""))
```

Comment:
`"ggm"` was added to calculate the `statistical` `AdjacencyMatrix`.
```
stat_adj <- statistical(mat_test[, -1],
    model = c("clr", "aracne", "pearson", "spearman", "ggm", "bayes"))
```
 When applying `threshold(stat_adj, type = "top2", args = list(n = 10))` the resulting `AdjacencyMatrix` is already different to the one in the previous version without ggm. Therefore also the combined `AdjacencyMatrix` will look a little bit different.

Reply:
I see, but I do not understand why the `structural` `AdjacencyMatrix` object will be different. This object should be identical to the object before your addition and not depend on the addition of the `ggm` model. The rows and columns should not be shifted/changed when combining the `structural` and `statistical` `AdjacencyMatrix` objects. It is true that the `statistical` `AdjacencyMatrix` object will be different, but there should be no "novel" transformation in the `combined` object - I expect that either the transformation of `statistical` will be kept or be removed by combining the `structural` and `statistical` objects, but there should be no transformations popping up that were not previously at this position.

Reply:
Oh my bad, you are refering to different rows here (changed from `[1,]` to `[5,]`. Is there any reason why you did that?

Reply:
Ah okay now I get where the confusion came from. So If I run 
`as.vector(assay(cons_adj, "combine_group")[1, ])`
if` "ggm"` was added to calculate the `statistical` my output is 
`[1] NA "" "" "" "" "" ""`
Therefore I selected a row (`[5,]`) where there was still an entry left. 
Does this make sense? 

So the `structural` `AdjacencyMatrix` did not change, it is just the thresholded/ combined that changed because of the `ggm` correlation values

Reply:
Perfect. Then that's fine for me :)